RecoTracker/SpecialSeedGenerators/python/cosmicDC_cff.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+# seeding
+from RecoMuon.CosmicMuonProducer.cosmicMuons_cfi import *
+import RecoMuon.MuonIdentification.muons1stStep_cfi
+muonsForCosmicDC = RecoMuon.MuonIdentification.muons1stStep_cfi.muons1stStep.clone(
+    inputCollectionLabels = cms.VInputTag("cosmicMuons"),
+    inputCollectionTypes = cms.vstring('outer tracks'),
+    fillIsolation = cms.bool(False),
```

Comment:
why are you not using muonsFromCosmic, which are already available in the standard sequence?
They differ from this definition in that they use dt4DCosmicSegments in the muonID variables definition.
So, muonsFromCosmic should give the same set of reco::Muons with the same kinematics, just different ID (matching and timing variables).

If there is no good reason to use this special instance, please switch to muonsFromCosmic.